src/mono/browser/debugger/BrowserDebugProxy/MonoSDBHelper.cs

```diff
@@ -525,7 +525,9 @@ public override string ReadString()
             {
                 data.Reverse();
             }
-            data.CopyTo(new Span<byte>(Unsafe.AsPointer(ref ret), data.Length));
+#pragma warning disable CS8500 // takes address of managed type
+            data.CopyTo(new Span<byte>(&ret, data.Length));
```

Comment:
Is the span constructor taking `ref` better here?

Reply:
I don't know if this has access to the internal constructor, and `ref` to what, since it's not a byte?

Reply:
`Unsafe.As` may be too verbose here. Since the addressing rule allows taking address it should be fine.